settings/Session.go

```diff
@@ -0,0 +1,6 @@
+package settings
+
+// Branding contains the branding settings of the app.
+type Session struct {
+	Timeout uint64 `json:"timeout"`
```

Comment:
Let's use `time.Duration` instead

Reply:
What do you mean?
You want the config to be like that 
```
./filebrowser config set --time.Duration
```
or what?

Reply:
No, I meant `./filebrowser config set 2h`. Check this link https://pkg.go.dev/time#ParseDuration